### ISSUE TITLE
Switch from ovatool for building OVAs to open-vmdk

### ISSUE THREAD

**Issue by dirkmueller** [OSInside/kiwi#2292]
## Problem description

kiwi currently uses ovatool for building vmware OVA images. there is an Apache-2.0 licensed project by VMware for this instead:

https://github.com/vmware/open-vmdk

I don't know how feature complete it is, but it would be nice to be able use it with kiwi and figure out any incompatibilities. OVA format is largely preferred by VMware users over the other option of vmdk. For users that can not build their images using the proprietary ovatool, open-vmdk is the only option.

## Expected behaviour

Stream-optimized OVA images can be built without dependency on proprietary tooling with kiwi.

## OS and Software information

Not relevant

**Comment by Conan-Kudo** [comment]
Oh thank god... It had been on my TODO list to gut the `ovatool` dependency, but if there's now something we can use, we should try this!

**Comment by abonillabeeche** [comment]
Who do we need to request this too?

**Comment by schaefi** [comment]
I agree it would be a good move. I remember there was the openovf project in the past and then we added code to use it and after some years it died and we ripped out everything we had in kiwi again and offered this "we call ovftool if present" proprietary binary thing. It's a solution but I think nobody is really happy with it.

So if open-vmdk can help here it would be great. Regarding resources, I believe at the moment it can only be looked at during the SUSE hackweek, I'm not aware of a direct volunteer you could assign this task to

**Comment by schaefi** [comment]
I'll put this on the agenda for the community meeting

**Comment by schaefi** [comment]
After a conversation and a short look into the code and the maturity of the project we decided to flag this as a kiwi-10 (next major kiwi release) feature and give open-vmdk some time to settle and see where it goes

**Comment by Conan-Kudo** [comment]
This is now landing in Fedora thanks to @ismail, so we can seriously consider this.

**Comment by schaefi** [comment]
Some work done from @gmoro 

* https://build.suse.de/projects/home:gmoro:ovacompose/packages/kiwi-open-vmdk/files/kiwi_post_run

It is soooo sad that Guilherme didn't had a chance to finish this into a PR here.

**Comment by johanneskastl** [comment]
Any news on this?

**Comment by johanneskastl** [comment]
Not stale, go away, stalebot... :-)
